src/main/java/Storage.java

```diff
@@ -0,0 +1,122 @@
+import java.io.File;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.PrintWriter;
+
+public class Storage {
+    String filePath;
+
+    /**
+     * Storage constructor.
```

Comment:
According to the coding standard, a line should be left between the description in a Javadoc comment and the @param/@return fields.